### PR TITLE
[WebDriver BiDi] `userPromptOpened` supports `defaultValue`

### PR DESCRIPTION
**Pull request by Lightning00Blade**
Issue on BiDi spec: https://github.com/w3c/webdriver-bidi/issues/521

### CODE REVIEW

File: webdriver/tests/bidi/browsing_context/user_prompt_opened/user_prompt_opened.py [hunk 1]
```diff
@@ -56,6 +56,28 @@ async def test_prompt_type(
     }
 
 
+async def test_prompt_default_value(bidi_session, subscribe_events, inline, new_tab,
+                           wait_for_event, prompt_type):
```
Comment (whimboo) [inline webdriver/tests/bidi/browsing_context/user_prompt_opened/user_prompt_opened.py:60]: The indentation is off. Maybe you can fix it? 

Also `prompt_type` is from a parameterization of another test and not used here. It should basically fail that test. Interesting that it doesn't.

```suggestion
async def test_prompt_default_value(
    bidi_session, inline, new_tab, subscribe_events, wait_for_event
):
```

**Review by whimboo — CHANGES_REQUESTED** [review]

**Comment by whimboo** [comment]
@Lightning00Blade as it looks like the [Chrome wpt.fyi results are failing](https://github.com/web-platform-tests/wpt/pull/41814/checks?check_run_id=16536380897) because of a test timeout and no results being reported. It doesn't seem to be a regression from this PR but please confirm. Thanks.

**Comment by whimboo** [comment]
Whereby [checking the real log shows that this newly added test fail](https://community-tc.services.mozilla.com/tasks/DHBE16NeRLi6q9NaMS7T_A/runs/0/logs/live/public/logs/live.log#L658-659)s and maybe as well causes the timeout?

File: webdriver/tests/bidi/browsing_context/user_prompt_opened/user_prompt_opened.py [hunk 2]
```diff
@@ -56,6 +56,29 @@ async def test_prompt_type(
...
+    await bidi_session.browsing_context.navigate(
+        context=new_tab["context"],
+        url=inline(f"<script>window.prompt('{text}', '{default}')</script>"),
+    )
+
+    event = await on_entry
+
+    assert event == {
+        "context": new_tab["context"],
+        "type": prompt_type,
```
Comment (lutien) [inline webdriver/tests/bidi/browsing_context/user_prompt_opened/user_prompt_opened.py:77]: `prompt_type` variable is not defined in this test, maybe that causes the failures.

**Review by whimboo — APPROVED** [review]
That made it! All tests are passing now. Thanks!

**Pull request merged**